pom.xml

```diff
@@ -102,6 +102,7 @@
     <jt.version>1.6.0</jt.version>
     <jaxb.api.version>2.4.0-b180830.0359</jaxb.api.version>
     <jaxb.runtime.version>2.4.0-b180830.0438</jaxb.runtime.version>
+    <javax.activation-api.version>1.2.0</javax.activation-api.version>
```

Comment:
Looks like we also use this version (hard-coded, rather than managed) at https://github.com/geotools/geotools/blob/master/modules/plugin/imagemosaic/pom.xml#L216

Not sure if you want to consolidate in this PR, or just pick it up later.

Reply:
I'll fix that now, thanks